src/test/java/org/broadinstitute/barclay/argparser/CommandLineArgumentParserTest.java

```diff
@@ -769,39 +780,34 @@ public void testWithBoundariesArgumentsForNoNumeric() {
     )
     public class WithBoundariesArguments {
 
-        @Argument(doc = "Double argument in the range [10, 20]", optional = true, minValue = 10, maxValue = 20)
+        @Argument(doc = "Double argument in the range [10, 20]", optional = true, minValue = 10, minRecommendedValue = 16, maxRecommendedValue = 17, maxValue = 20)
```

Comment:
Can you add a comment to these saying that the recommended values here are not explicitly verified by the tests, but do force the code through the warning code paths.